src/global.js

```diff
@@ -240,12 +240,14 @@ export const GlobalStyles = createGlobalStyle`
     .dapp-nav a {
         border-bottom: 2px solid transparent;
     }
-    .dapp-nav a:hover {
-        > span {
-            border-bottom: 2px solid ${({ theme }) => theme.color};
+    .dapp-nav a{
+        &:hover {
+            > span {
+                border-bottom: 2px solid ${({ theme }) => theme.color};
+            }
         }
         &.active span {
-            border-bottom: 2px solid ${({ theme }) => theme.activeLinkColor};
+            border-bottom: 2px solid ${({ theme }) => theme.activeLinkColor} !important;
```

Comment:
`!important`s  can get messy especially with layered styles. anyway we can do this without it?